### ISSUE TITLE
Mystical error message when trying to start docker-sync

### ISSUE THREAD

**Issue by Nomafin** [EugenMayer/docker-sync#485]
I get the following error when I try to start docker-sync for any of my projects. Same projects worked for me before and work still for my co-workers with similar setups. Just can't figure out what has changed.

### Error/Feature Requestion/Docs
```
$ docker-sync start
      ok  Starting unison for sync project-sync
      ok  project-sync container not running
      ok  starting project-sync container
 command  docker start project-sync && docker exec project-sync supervisorctl restart unison
      ok  starting initial sync of project-sync
 command  docker inspect --format='{{(index (index .NetworkSettings.Ports "5000/tcp") 0).HostPort}}' project-sync
 command  unison -testserver /app_sync "socket://:32777"
/Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/lib/docker-sync/sync_strategy/unison.rb:191:in `block in start_container': Failed to start unison container in time, try to increase max_attempt in your configuration. See https://github.com/EugenMayer/docker-sync/wiki/2.-Configuration for more informations (RuntimeError)
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/lib/docker-sync/sync_strategy/unison.rb:187:in `loop'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/lib/docker-sync/sync_strategy/unison.rb:187:in `start_container'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/lib/docker-sync/sync_process.rb:107:in `start_container'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/lib/docker-sync/sync_manager.rb:92:in `block in start_container'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/lib/docker-sync/sync_manager.rb:91:in `each'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/lib/docker-sync/sync_manager.rb:91:in `start_container'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/tasks/sync/sync.thor:166:in `daemonize'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/tasks/sync/sync.thor:48:in `start'
from /Library/Ruby/Gems/2.0.0/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
from /Library/Ruby/Gems/2.0.0/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
from /Library/Ruby/Gems/2.0.0/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
from /Library/Ruby/Gems/2.0.0/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
from /Library/Ruby/Gems/2.0.0/gems/docker-sync-0.4.6/bin/docker-sync:14:in `<top (required)>'
from /usr/local/bin/docker-sync:23:in `load'
from /usr/local/bin/docker-sync:23:in `<main>'
```

### Docker Driver
Docker for Mac

### Sync strategy
native_osx

### your docker-sync.yml
```
version: "2"

options:
  verbose: true
syncs:
  project-sync:
    src: '.'
    host_disk_mount_mode: 'cached'
```

### OS
OS X Sierra 10.12.6

**Comment by EugenMayer** [comment]
you are using docker for mac `edge` which is incompatible with `0.4.6`, please use `0.5.0-beta2`

**Comment by rajata07** [comment]
@EugenMayer I have the same problem. How can i install 0.5.0-beta2? I tried already `gem install docker-sync -v 0.5.0-beta2`

**Comment by jerome-arzel** [comment]
Try `gem install docker-sync -v '~> 0.5.0'`.

**Comment by EugenMayer** [comment]
guys, docker-sync is released already, 0.5.2 is on the list. So why using the beta?

**Comment by jerome-arzel** [comment]
My command will download the last release. And I guess that is because this issue is old.
